tests/management/role/test_view.py

```diff
@@ -256,48 +286,7 @@ def test_read_role_list_success(self):
                 self.assertEqual(iterRole.get("accessCount"), 1)
                 role = iterRole
         self.assertEqual(role.get("name"), role_name)
-
-    def test_get_role_by_partial_name_by_default(self):
```

Comment:
Are these deleted by accident?

Reply:
Oh, yep. Not sure if a rebase thing or a different issue, but I'll get that resolved here in a sec. Thanks!